controlpanel/frontend/jinja2/tools.html

```diff
@@ -12,56 +12,45 @@ <h1 class="govuk-heading-xl">Your tools</h1>
 <table class="govuk-table">
```

Comment:
FYI: I tried to run this locally. Not sure why yet but I can't see anything in the tools table (it says `0 tools`) - helm releases are there and pods are running 🤷‍♂

Reply:
Did you run the migrations?

Reply:
I didn't before. After running them it's all good ✨  😍